docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   terracost:
-    image: cycloid/golang:1.15.6
+    image: golang:1.17.3
```

Comment:
It's weird to use `1.17.3` here but `go 1.16` on the go.mod isn't it?

Reply:
The mod is minimum version to run it (as we use some specific new methods from 1.16) but the image version can be any one higher than that one, that's why I used the latest one as of now.